Java/Searching/Binary Search/BinarySearch.java

```diff
@@ -0,0 +1,25 @@
+public class BinarySearch {
```

Comment:
Add the comment on the top of the file related to the program and what problem it solves.

Like in this case you can add:
`Binary Search Algorithm Implementation`